src/containers/ServiceWorker/messages.ts

```diff
@@ -0,0 +1,19 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  newVersionTitle: {
+    defaultMessage: "System update required",
+    description: "notification title"
+  },
+  newVersionContent: {
+    defaultMessage:
+      "You need to update your system to the newest version. Before doing so - save all  changes to prevent losing that data. To update use the button below.",
```

Comment:
>system

This sounds a bit similar to the OS update and might confuse some users. Perhaps "You need to update Saleor to the newest version"? cc @pwgryglak 

```suggestion
      "You need to update your system to the newest version. Before doing so - save all changes to prevent losing that data. To update use the button below.",
```
double whitespace

Reply:
Sure sounds like a good idea

Reply:
:+1:

Reply:
Also I believe as I read that through once again it should say something more like this: "You need to update Saleor to the newest version. Before doing so - save all  changes to prevent loss of data.
To update use the button below." I think it's more gramatically correct